.storybook/documentation.scss

```diff
@@ -137,3 +137,36 @@ html {
     line-height: 19px;
   }
 }
+
+// Company Card
+.card-company {
```

Comment:
Note: This is only in the documentation. It does not affect the published package.